### DISCUSSION TITLE
Schedule to do not being banned from speedtest

### DISCUSSION THREAD

**Discussion by ryosoftware** [alexjustesen/speedtest-tracker#1584] [Q&A]
Hello,
Is there a know value for schedule to maximize the stats but not being banned from Ookla's Speedtest Service?
Thanks

**Comment by svenvg93** [comment]
There isnt really a know value when you will be blocked. 
From a network engineer perspective, I would advice not to go lower then once a hour. Otherwise it will impact your own internet usage and your internet usage will implact the speedtest. My sweet spot so far is every 2 hours. As it give it a good inidication for problems with the line it self, and congestion within the isp network.
